styles/main.css

```diff
@@ -565,11 +565,6 @@ label {
     text-decoration: none;
 }
 
-.state-dropdown-small{
```

Comment:
Instead of removing this class, would it be possible to change it to something like
```
.state-dropdown--large {
    max-height: 323px;
}
``` 
but only in the `(min-width: 960px)` media query, so that it will only apply when there are two columns displayed? That way the two columns will still align on large screens, and users will be able to see more states and won't need to scroll as much.